code/modules/food_and_drinks/food.dm

```diff
@@ -5,6 +5,8 @@
 /// the parent to the exclusion list in code/__HELPERS/unsorted.dm's
 /// get_random_food proc.
 ////////////////////////////////////////////////////////////////////////////////
+#define STOP_SERVING_BREAKFAST 15 MINUTES
```

Comment:
```suggestion
#define STOP_SERVING_BREAKFAST (15 MINUTES)
```

Reply:
done, does it not read it correctly without the parenthesis or something?

Reply:
`15 MINUTES` expands to `15 * 60 * 10` so there is a possible order of operations risk. It's bitten us before.